<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListaBadan</class>
 <widget class="QWidget" name="ListaBadan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame_tab">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_badanie">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="OdtwarzalnoscWyniki" name="pOdtwarzalnosc">
         <layout class="QFormLayout" name="formLayout"/>
        </widget>
        <widget class="PowtarzalnoscWyniki" name="pPowtarzalnosc">
         <layout class="QFormLayout" name="formLayout_2"/>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>NIe zdefiniowana</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OdtwarzalnoscWyniki</class>
   <extends>QWidget</extends>
   <header>odtwarzalnoscwyniki.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PowtarzalnoscWyniki</class>
   <extends>QWidget</extends>
   <header>powtarzalnoscwyniki.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
